src/lib/unzip.js

```diff
@@ -4,13 +4,12 @@ import path from 'path';
 import win7zip from 'win-7zip';
 const isDevEnv = process.env.NODE_ENV === 'development';
 
+const pathTo7zip = process.platform === 'win32' ? win7zip['7z'] : path7za;
+
 // https://github.com/puppeteer/puppeteer/issues/2134#issuecomment-408221446
-const pathTo7zipCross = isDevEnv
-  ? path7za
-  : path7za.replace('app.asar', 'app.asar.unpacked');
-const pathTo7zipWin = isDevEnv
-  ? win7zip['7z']
-  : win7zip['7z'].replace('app.asar', 'app.asar.unpacked');
+if (!isDevEnv) {
+  pathTo7zip.replace('app.asar', 'app.asar.unpacked');
```

Comment:
`String.replace()` does not change the object.
It just returns a new string.
```suggestion
  pathTo7zip = pathTo7zip.replace('app.asar', 'app.asar.unpacked');
```

Reply:
Sorry for this stupid mistake. I fixed the problem. I will pay more attention next time.

Reply:
Please do not say such a thing.
We, everyone, could make a mistake.
It is OK.